Test/AbstractWidgetTestCase.php

```diff
@@ -172,4 +172,20 @@ protected function getRenderingEngine()
             return preg_replace('~\s*~', '', $m[0]);
         }, $html);
     }
+
+    /**
+     * NEXT_MAJOR: Remove this method when dropping support for < PHPUnit 5.4.
+     *
+     * @param string $class
+     *
+     * @return \PHPUnit_Framework_MockObject_MockObject
+     */
+    protected function createMock($class)
```

Comment:
👎 because the original `createMock` method isn't final too: https://github.com/sebastianbergmann/phpunit/blob/5.4/src/Framework/TestCase.php#L1526

This would irritate someone.